### DISCUSSION TITLE
How to view the outputs of a workflow without using UI

### DISCUSSION THREAD

**Discussion by vidhi-pavan-agrawal** [Netflix/conductor#2525] [Q&A] [answered]
hello everyone, 
basically, I have written a dynamic workflow I want to return the output of the workflow in JSON, any suggestions are welcomed. Thanks in advance.

**Comment by peterlau** [comment, marked as answer]
The UI simply utilizes the published API - you should be able to poll for the current state of a running workflow with its ID using the `/workflow/{workflowID}` endpoint. The input and output of the workflow itself (as well as any invoked tasks) are part of the payload.
